### PR TITLE
Add support for Hugging Face Speech2TextForConditionalGeneration models

### PR DESCRIPTION
**Pull request by aaron-siegel**

### CODE REVIEW

**Comment by aaron-siegel** [comment]
This change is [Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/362)

File: pixeltable/functions/huggingface.py [hunk 1]
```diff
@@ -362,6 +363,86 @@ def vit_for_image_classification(
...
+    if language is not None and language not in processor.tokenizer.lang_code_to_id:
+        raise excs.Error(
+            f"Language code '{language}' is not supported by the model '{model_id}'. "
+            f"Supported languages are: {list(processor.tokenizer.lang_code_to_id.keys())}")
+
+    forced_bos_token_id: Optional[int] = None if language is None else processor.tokenizer.lang_code_to_id[language]
+
+    # Get the model's sampling rate. Default to 16 kHz (the standard) if not in config
+    model_sampling_rate = getattr(model.config, 'sampling_rate', 16_000)
+    waveform, sampling_rate = librosa.load(audio, sr=model_sampling_rate, mono=True)
```
Comment (mkornacker) [inline pixeltable/functions/huggingface.py:432]: What does this produce? Is this an array? In general, it's problematic to combine IO and computation utilizing multiple resource in a single udf.

Comment (aaron-siegel) [reply pixeltable/functions/huggingface.py:432]: As discussed today - I agree in principle, but until we have parallelized execution there is no user-observable difference in doing it this way, and `@expr_udf` isn't yet ready to be a replacement for `@udf` (see PXT-344).

**Review by aaron-siegel — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/362)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @mkornacker)

**Review by mkornacker — APPROVED** [review]

**Pull request merged**